niworkflows/interfaces/masks.py

```diff
@@ -189,11 +189,11 @@ def _post_run_hook(self, runtime):
         ''' generates a report showing slices from each axis '''
 
         self._anat_file = self.inputs.realigned_file
-        self._mask_file = self.aggregate_outputs().high_variance_mask
-        self._seg_files = [self.aggregate_outputs().high_variance_mask]
+        self._mask_file = self.aggregate_outputs().high_variance_masks
+        self._seg_files = [self.aggregate_outputs().high_variance_masks]
```

Comment:
The problem is that `high_variance_masks` is an OutputMultiPath so it converts single element lists to just the inner element. We could alternatively run a function on it to convert it back to a list.